sbt-version-policy/src/test/scala/testpkg/VersionChangeTest.scala

```diff
@@ -0,0 +1,37 @@
+package testpkg
+
+import coursier.version.VersionCompatibility._
+import sbtversionpolicy.VersionChange
+import verify._
+
+object VersionChangeTest extends BasicTestSuite {
+  test("SemVer Spec") {
+    assert(VersionChange("0.1.0", "0.1.0", SemVerSpec) == VersionChange.Same)
+    assert(VersionChange("0.1.0", "0.1.1", SemVerSpec) == VersionChange.Prerelease)
+    assert(VersionChange("0.1.0", "0.2.0", SemVerSpec) == VersionChange.Prerelease)
```

Comment:
Same here.

Reply:
Addressed in https://github.com/scalacenter/sbt-version-policy/pull/31/commits/db10da5f6647fdd12a61bdd0d3ab41133102e5cb